pom.xml

```diff
@@ -570,12 +570,6 @@
         <artifactId>commons-beanutils</artifactId>
         <version>1.9.4</version>
       </dependency>
-      <dependency>
-        <groupId>commons-logging</groupId>
-        <artifactId>commons-logging</artifactId>
-        <!-- Hive uses commons-logging 1.1.3 from 0.13 to 1.2 -->
-        <version>1.1.3</version>
```

Comment:
Could you manual test if we can connect Hive 0.13 - Hive-1.2 after removing commons-logging?

Reply:
> Could you manual test if we can connect Hive 0.13 - Hive-1.2 after removing commons-logging?

Yea, our env is hive 1.2.1, I will test this later.

Reply:
> Could you manual test if we can connect Hive 0.13 - Hive-1.2 after removing commons-logging?

Hive need 0.13~ 1.2 need `commons-logging`.  but  `jcl-over-slf4j` can replace it. For different usage of hive jars:

1. builtin: not supported yet. 
2. maven: will download commons-logging-1.1.13, but only this jar in hive client 's  classLoader, not a problem since `jcl-over-slf4j`  and  `commons-logging` not coexistence.
3. path/classpath:   user's behavior, we can't  control, but better let user know this problem.

So it's safe to remove this.